tests/e2e/specs/merchant/checkout-terms.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	setCheckbox,
+	unsetCheckbox,
+} from '@woocommerce/e2e-utils';
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+import {
+	BILLING_DETAILS,
+	SIMPLE_VIRTUAL_PRODUCT_NAME,
+} from '../../../utils/constants';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `Skipping checkout tests`, () => {} );
+}
+
+describe( 'Merchant → Checkout → Can adjust T&S and Privacy Policy options', () => {
+	beforeAll( async () => {
+		await shopper.goToShop();
+		await page.goto( 'http://localhost:8889/?setup_terms_and_privacy' );
```

Comment:
```suggestion
		await page.goto( `${ process.env.WORDPRESS_BASE_URL }/?setup_terms_and_privacy` );
```
We can use our base URL instead (if we want to use the plugin to create pages for the test)

Reply:
See previous comment.